src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+const rows = table.querySelectorAll('tbody tr');
+
+rows.forEach(row => {
+  const secondCell = row.cells[1];
+
+  const clonedCell = secondCell.cloneNode(true);
+
+  row.insertBefore(clonedCell, row.cells[row.cells.length - 1]);
+});
+
+const headerRow = table.querySelector('thead tr');
+const newHeaderCell = document.createElement('th');
+const copyCellName = Array.from(headerRow.querySelectorAll('th'));
```

Comment:
since we don't need the whol array of sell here but only the second one you can get it. `querySelector` accepts any type of selectors, for example `th:nth-child(2)`